src/Navigation.re

```diff
@@ -14,6 +24,16 @@ external navigateWithParams: (t, string, Js.t({..})) => unit = "navigate";
 [@bs.send] external goBackToRoute: (t, string) => unit = "";
 [@bs.send] external getParam: (t, string) => Js.nullable('a) = "";
 [@bs.send] external getParamWithDefault: (t, string, 'a) => 'a = "getParam";
+[@bs.send]
+external addListener:
+  (
+    t,
+    [@bs.string] [ | `didBlur | `didFocus | `willBlur | `willFocus],
+    navigationEventPayload => unit
+  ) =>
+  EventSubscription.t =
+  "addListener";
+
 // TODO: addListener, setParams, ...
```

Comment:
I just noticed one last thing now, sorry! Remove "addListener" from the TODO. 😉

Everything else looks fine now! 👍